modules/dnn/src/net_openvino.cpp

```diff
@@ -864,15 +804,8 @@ Net openvino_readNetwork(
     InferenceEngine::CNNNetwork ieNet;
```

Comment:
it's removed in API 2.0

Reply:
This is a workaround to wrap `ov::Model` from API 2.0: https://github.com/opencv/opencv/pull/25199/files#r1524623439